docker-compose.yml

```diff
@@ -17,6 +17,8 @@ services:
       - ALLOW_ALL_IFRAMES=${ALLOW_ALL_IFRAMES:-False}
       - ALLOWED_HOSTS=*
       - ATTACHMENT_HOST=${ATTACHMENT_HOST:-localhost:8000}
+      - AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID:-}
+      - AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY:-}
```

Comment:
It appears, unless I've gone crazy that `docker-compose.yml` will automagically read this file. So this line (and line above and line 33) are not needed.